scene/gui/range.h

```diff
@@ -100,6 +100,8 @@ class Range : public Control {
 	void set_allow_lesser(bool p_allow);
 	bool is_lesser_allowed() const;
 
+	bool allow_non_finite = false;
```

Comment:
This shouldn't be a public member variable, instead it should have setters and getters